src/Evolution/Executables/GeneralizedHarmonic/CMakeLists.txt

```diff
@@ -73,3 +73,11 @@ add_spectre_parallel_executable(
   "EvolutionMetavars"
   "${LIBS_TO_LINK};ControlSystem;GeneralRelativitySolutions;Importers"
 )
+
+add_spectre_parallel_executable(
```

Comment:
I'm ok with this as is, but how hard would it be to just use a template parameter to EvolveGhBinaryBlackHole.hpp specifying whether or not we use CCE? Would this minimize code duplication enough to make it worth it?

Reply:
I think I could make it work, but the main reason I opted not to do this is because then you'd have to have some CCE stuff in the metavariables even if you didn't use it. And the BBH+CCE executable still takes longer than just BBH to compile so I'd be worried that you'd have to compile CCE stuff even if you only wanted BBH. Until we figure out how to reduce compile-time resource usage for BBH+CCE, I think it's best to leave them separate.